### ISSUE TITLE
PlayerMoveEvent doesn't gets called

### ISSUE THREAD

**Issue by MarvinKlar** [MyPetORG/MyPet#1523]
# Bug report

* [X] I have checked other issues to make sure this is not a duplicate.

## Describe the bug

If I enable the setting `MyPet.Throw-PlayerMoveEvent-While-Riding` (by setting it to `true`) the PlayerMoveEvent still doesn't gets called.

## To Reproduce

1. Enable `Throw-PlayerMoveEvent-While-Riding` in the `config.yml`.
2. Restart the server
3. Ride a Horse

## Expected behavior

The PlayerMoveEvent should get called.

**Bukkit/Spigot/Bungee Build/Version:**
![grafik](https://user-images.githubusercontent.com/34421947/154934620-7f8aa275-037a-4ca6-a75c-6c2b10d480de.png)

**MyPet Build/Version:**
Latest (3.12-SNAPSHOT-b)

**Comment by MarvinKlar** [comment]
I just had a look´in the code and saw this:
```
// throw player move event
if (Configuration.Misc.THROW_PLAYER_MOVE_EVENT_WHILE_RIDING && !(this instanceof EntityMyHorse)) {
```

Can somebody explain, why the event is not thrown for horses?

**Comment by MarvinKlar** [comment]
If I remove `!(this instanceof EntityMyHorse)` everything works fine.

**Comment by MarvinKlar** [comment]
@Jakllp Any idea?

**Comment by Jakllp** [comment]
Same reason for the bug in https://github.com/MyPetORG/MyPet/issues/1181 -> for some reason riding horses (and some other mobs that are "rideable" without commands) is handled not server but client side hence the desync.

The desync is fixed since `3.12-SNAPSHOT-B1666`

So it probably caused problems to throw this event before that build.

Consider this basically fixed (gonna check it with my next push)

**Comment by MarvinKlar** [comment]
Thank you!
